src/sickle.ts

```diff
@@ -263,6 +267,17 @@ class Annotator {
     this.emit(': void;\n');
   }
 
+  private visitEnum(node: ts.EnumDeclaration) {
+    this.emit('\n');
+    for (let member of node.members) {
+      this.emit('/** @export */ ');
```

Comment:
Would it make sense to only @export these if the enum itself is @exported or something?  I'm imagining a hypothetical size-sensitive codebase where we haven't provided any knob to opt out of this.